factor/capability/lang/en/factor_capability.php

```diff
@@ -0,0 +1,31 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Language strings.
+ *
+ * @package     factor_capability
+ * @author      Peter Burnett <peterburnett@catalyst-au.net>
+ * @copyright   Catalyst IT
+ * @license     http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+$string['pluginname'] = 'User capability';
+$string['privacy:metadata'] = 'The  user capability factor plugin does not store any personal data';
+$string['settings:adminpasses'] = 'Site admins pass this factor';
+$string['settings:adminpasses_help'] = 'Controls whether site administrators should be able to pass this factor without an explicit role binding.';
```

Comment:
I might have got the language here the wrong way around, but I think the help could flesh this out something like:

By default admins pass all capability checks, which means they will fail this factor.

If checked then all site admins and anyone without this capability will pass.

If not checked the only site admins without this capability will pass.